### ISSUE TITLE
does not work with APIs versioning

### ISSUE THREAD

**Issue by piggyman007** [fmvilas/swagger-node-codegen#41]
Hi,

I found that this lib does not work with APIs versioning, e.g., my APIs may have
- /api/v1/users
- /api/v2/users

I would like to create 2 more folders, v1 and v2 under routes and services.
But It is impossible to do it for this version.

**Comment by fmvilas** [comment]
You're right. It's a missing feature. Would you mind adding it? It would be very helpful.

**Comment by smithmr** [comment]
+1
